cl/lib/redis_utils.py

```diff
@@ -76,3 +76,24 @@ def delete_redis_semaphore(r: Union[str, Redis], key: str) -> None:
     if isinstance(r, str):
         r = make_redis_interface(r)
     r.delete(key)
+
+
+def check_or_create_email_sending_quota(r: Redis) -> None:
```

Comment:
This also doesn't belong in redis_utils.py. It should probably just live with the email backend, since that's all it's going to be used with.